Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/requirements.txt

```
@@ -1,18 +1,19 @@
-torch==1.7.1+cpu
+# both conda and requirements
 mlflow>=1.12.0
-torchserve
-torch-model-archiver
-numpy
-torchserve
-torch-model-archiver
-numpy
-scikit-learn
-cloudpickle==1.6.0
-boto3
+sklearn
 transformers
 torchtext
 torch
-torchvision
 pandas
-sklearn
+# conda only -> needed for training?
+cloudpickle==1.6.0
+boto3
+torchvision
```

Same as torch package. Keeping CPU version meanwhile we have no GPUs to test on.
```suggestion
torchvision==0.8.2+cpu
```

---

Will be resolved by base image using serve install_dependencies